detection/notebooks/00_webcam.ipynb

```diff
@@ -0,0 +1,650 @@
+{
```

Comment:
<ul><li>It says here that "we need to convert" but looking below there is no implicit conversion (but I guess the T.Compose(T.ToTensor)(im)) does it? That's a bit confusing. Maybe instead write "The model expects RGB images as input. Below, the RGB image is converted to a torch.FloatTensor which is then passed in to the model"</li><li>"The output of the model is a probability distribution of the <a href="https://github.com/pytorch/vision/blob/master/docs/source/models.rst#object-detection-instance-segmentation-and-person-keypoint-detection" target="_blank">labeled classes</a>" don't think that's true since for each rectangle it only outputs the label with maximum score. I think what you want to say is what classes the model was trained on and hence can detect. How about remove that sentence here, and then add in the "Load Pretrained Model" after the sentence "The <a href="https://pytorch.org/docs/stable/torchvision/models.html#object-detection-instance-segmentation-and-person-keypoint-detection" target="_blank">reported box AP</a> is 37.0 trained on the instances set of COCO train2017 and evaluated on COCO val2017. ": "Coco is a popular dataset in the research community and consists of XX images with in total XX annotated objects of XX distinct classes LINK HERE such as person, car, boat, or cup."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Yes.  <code>T.Compose(T.ToTensor)(im))</code> carries out the conversion from (H, W, C) in [0, 255] to (C, H, W) in [0.0, 0.1].</p>